src/System.Threading.Tasks.Extensions/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0008</DiagnosticId>
+    <Target>T:System.Runtime.CompilerServices.AsyncMethodBuilderAttribute</Target>
+    <Left>lib/net461/System.Threading.Tasks.Extensions.dll</Left>
+    <Right>lib/netstandard2.0/System.Threading.Tasks.Extensions.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETPortable,Version=v0.0,Profile=Profile259</Target>
```

Comment:
I updated this PR with the latest changes in main, which contain the suppresion of netstandard1.0 in eng/packaging.targets. I removed netstandard1.0 from here and I can confirm that `dotnet pack` (with IsPackable=true) passes:

```
dotnet pack
MSBuild version 17.8.3+195e7f5a3 for .NET
  Determining projects to restore...
  Restored D:\maintenance-packages\src\System.Threading.Tasks.Extensions\src\System.Threading.Tasks.Extensions.csproj (in 259 ms).
  System.Threading.Tasks.Extensions -> D:\maintenance-packages\artifacts\bin\System.Threading.Tasks.Extensions\Release\net462\System.Threading.Tasks.Ext
  ensions.dll
  System.Threading.Tasks.Extensions -> D:\maintenance-packages\artifacts\bin\System.Threading.Tasks.Extensions\Release\netstandard2.0\System.Threading.T
  asks.Extensions.dll
  The package System.Threading.Tasks.Extensions.4.6.0-dev.24081.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to lea
  rn why package readmes are important.
  Successfully created package 'D:\maintenance-packages\artifacts\packages\Release\Shipping\System.Threading.Tasks.Extensions.4.6.0-dev.24081.1.nupkg'.
```

Since `.NETPortable,Version=v0.0,Profile=Profile259` needs https://github.com/dotnet/sdk/pull/38374 to be merged and then flowed to this repo, I cannot yet remove it. We can address that specific suppresion removal after we get that sdk change flowed here, so that we don't block merging this PR. Is that ok, @ViktorHofer ?

Reply:
> Since .NETPortable,Version=v0.0,Profile=Profile259 needs https://github.com/dotnet/sdk/pull/38374 to be merged

That PR will just make things easier as it supports wildcards but you should be able to suppress that specific TFM alias already today. Make sure that what you pass in is the expected format that is returned by NuGetFramework.GetShortFolderName().

Reply:
I submitted an additional PR with a bunch of ignorable tfms: https://github.com/dotnet/maintenance-packages/pull/53

. I included the one that you just mentioned, and I double checked that it was indeed the right value:
```cs
using NuGet.Frameworks;
using System;
internal class Program
{
    private static void Main(string[] args)
    {
        var framework = NuGetFramework.Parse(".NETPortable,Version=v0.0,Profile=Profile259");
        var shortname = framework.GetShortFolderName();
        Console.WriteLine(shortname); // prints: portable-net45+win8+wp8+wpa81
    }
}
```